src/css/ttyg/agent-settings-modal.css

```diff
@@ -52,7 +47,7 @@
     width: 100%
 }
 
-.agent-settings-form .additional-extraction-methods .agent-settings-form [type="range"] {
+.agent-settings-form [type="range"] {
```

Comment:
Not all range fields were designed this way. Check if you didn't brake something with such broad selector!

Reply:
We have three range inputs: 'Similarity Threshold', 'Top_P', and 'Temperature'. All of them should have this styling, but all of them have an ID, so I can use it to create a more specific selector.

Reply:
I saw that you fixed this in your MR, so I reverted the changes.